src/test/scala/uk/gov/hmrc/perftests/example/TrackingConsentFrontendSimulation.scala

```diff
@@ -21,7 +21,8 @@ import uk.gov.hmrc.perftests.example.TrackingConsentFrontendRequests._
 
 class TrackingConsentFrontendSimulation extends PerformanceTestRunner {
 
-  setup("download", "Retrieve the tracking script") withRequests navigateToTrackingJs
+  setup("download", "Retrieve the tracking script") withRequests requestTrackingJs
```

Comment:
I think this is okay, I'm just wondering if the audit should be part of a journey where it also retrieves the script?